go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	k8s.io/apimachinery v0.17.1
 	k8s.io/client-go v0.17.1
 	k8s.io/klog/v2 v2.0.0
+	rsc.io/quote/v3 v3.1.0 // indirect
```

Comment:
I believe this means we need to ensure we are including the https://pkg.go.dev/rsc.io/quote/v3?tab=licenses.

Reply:
where should be put the license?

Reply:
Ideally we should have a top level directory LICENSES and a script which populated the directory for things like this. For now lets file a bug and I'll see if we can add that easily as a follow on.

Reply:
will file an issue.